src/interface.rs

```diff
@@ -74,6 +77,18 @@ impl VoicemeeterRemote {
                 .clone(),
         }
     }
+
+    /// Update the current program type.
+    pub fn update_program(&mut self) -> Result<(), GetVoicemeeterInformationError> {
```

Comment:
since this update method is is mutating, it requires that whatever variable is holding it be also mutable, problematic for custom types holding a remote inside

Reply:
> Alternatively methods like .parameters() could call the update themselves so that the info is up to date

actually scratch that, that would force .parameters() to be mutating too

Reply:
> At least the compile time error that a field with that name doesn't exist and instead there's a method with the same name would be both explicit and super easy to fix

actually the field could be left in place with a deprecation warning added on it, that way people's old code would still work

Reply:
I don't want to get the program type every time a call to script is made, I'm not sure how to handle this so I just created `update_program` to let the user handle what they want to do with what should happen when the user opens another voicemeeter.

I don't consider the breaking change a big deal, the library is still in pre-1.0 so breakage can happen in `0.x`. I use [cargo semver-checks](https://github.com/obi1kenobi/cargo-semver-checks) to ensure I properly version